client/js/lounge.js

```diff
@@ -49,6 +49,7 @@ $(function() {
 
 		slideoutMenu.toggle(isOpen);
 		storeSidebarVisibility("thelounge.state.sidebar", isOpen);
+		utils.togglePreviewMoreButtonsIfNeeded();
```

Comment:
Perhaps move both of these calls into `storeSidebarVisibility`? That also removes the call on mobile since it's an overlay there.

Reply:
Will try, thanks!